packages/components/src/heading/stories/index.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { Heading } from '..';
+
+const meta: ComponentMeta< typeof Heading > = {
+	component: Heading,
+	title: 'Components (Experimental)/Heading',
+	argTypes: {
+		adjustLineHeightForInnerControls: { control: { type: 'text' } },
+		as: { control: { type: 'text' } },
+		color: { control: { type: 'color' } },
+		display: { control: { type: 'text' } },
+		letterSpacing: { control: { type: 'text' } },
+		lineHeight: { control: { type: 'text' } },
```

Comment:
These values can be both strings or numbers, is there a way to achieve that with controls?

Reply:
The `object` control can strictly cover both, since that's basically a JSON string input so it differentiates between `1` and `"1"`. Though in the vast majority of cases I think a normal `text` control would be sufficient for tinkering and better for UX (because people will have trouble recognizing that they need to quote their strings in a `object` control).